ospatch/yum_update.go

```diff
@@ -78,6 +78,14 @@ func YumDryRun(dryrun bool) YumUpdateOption {
 	}
 }
 
+func packageNameWithArch(pkg *packages.PkgInfo) string {
+	nameWithArch := pkg.Name
```

Comment:
Small note about name of the function, as function is not always returns package with architecture, maybe just rename it to packageNameFrom. But i'm not insisting on that